docs/5.0/application-access.md

```diff
@@ -0,0 +1,264 @@
+
+## Example Applications
+
+As outlined in our introduction Teleport Application Access has been designed to support two types of applications.
+
+**Example Legacy App**</br>
+A device such as an load balancer might come with a control panel, but it doesn't' have any auth and can only be access via a privileged network. These applications are supported and can extend access beyond your network.
+
+Other example legacy apps.
+
++ An internal admin tool
++ Control panel for networking devices
+
+**Example Modern App**</br>
+Teleport Application Access supports all modern applications, these could be built in-house or off-the-shelf software such as jenkins, Kubernetes Dashboard and Jupyter workbooks.
+
++ Kubernetes Internal Dashboard
++ Grafana
++ Jupyter notebooks
++ In-house Single Page Apps (SPA)
++ SPA with custom Json Web Token support (JWT)
+
+## Teleport Application Service Setup
+
+Teleport
+**Define `/etc/teleport.yaml`**
+
+| Variable to replace | Description  |
+|-|-|
+| `nodename` | Name of node Teleport is running on |
+| `auth_token` | Static Join Token |
+| `public_addr` | Public URL and Port for Teleport |
+| `https_key_file` | LetsEncrypt Key File ( Wildcard Cert )  |
+| `https_cert_file` | LetsEncrypt Key File ( Wildcard Cert ) |
+
+
+```
+teleport:
+  nodename: i-083e63d0daecd1315
+  data_dir: /var/lib/teleport
+  auth_token: 4c7e15
+  auth_servers:
+  - 127.0.0.1:3025
+auth_service:
+  enabled: "yes"
+  listen_addr: 0.0.0.0:3025
+  tokens:
+  - proxy,node,app:4c7e15
+ssh_service:
+  enabled: "false"
+proxy_service:
+  enabled: "yes"
+  listen_addr: 0.0.0.0:3023
+  web_listen_addr: 0.0.0.0:3080
+  tunnel_listen_addr: 0.0.0.0:3024
+  public_addr: teleport.asteroid.earth:3080
+  ## Example using a wildcard cert.
+  https_keypairs:
+  https_keypairs:
+  - key_file: /etc/letsencrypt/live/teleport.example.com/privkey.pem
+  - cert_file: /etc/letsencrypt/live/teleport.example.com/fullchain.pem
+  - key_file: /etc/letsencrypt/live/*.teleport.example.com/privkey.pem
+  - cert_file: /etc/letsencrypt/live/*.teleport.example.com/fullchain.pem
+```
+#### [ Optional ] Obtain a new App Token
+In the above example we've hard coded a join token 4c7e15. You can use this to join apps or create a dynamic token using the tctl command below.
+```bash
+$ tctl tokens add --type=app
+```
+
+| Variable to replace | Description  |
+|-|-|
+| `auth_servers` | Address of Auth or Proxy Service setup above |
+| `auth_token` | Token used to connect to other Teleport processes  |
+
+```yaml
+teleport:
+  # nodename of machine running process.
+  nodename: wellington
+  # The Auth token `tctl tokens add --type=app` or the static token. 4c7e15
+  auth_token: "4c7e15"
+  # This is the location of the Teleport Auth Server or Public Proxy
+  auth_servers:
+    - teleport.example.com:3080
+auth_service:
+  enabled: no
+proxy_service:
+  enabled: no
+ssh_service:
+   enabled: no
+# The app_service is new
+app_service:
+   enabled: yes
+    debug_app: true
+   apps:
```

Comment:
may be split this into two parts. The first one with the debug app so they can quickly test and verify. Show the screenshot of what to expect. Second part with connecting k8s dashboard + config so they can edit and restart.